src/Script/Composer/Composer.php

```diff
@@ -92,7 +89,7 @@ private static function removeServiceDirs(
             }
         }
         $event->getIO()->write(
-            "Removed $deleteCount AWS services"
+            "Removed $deleteCount AWS service" . ($deleteCount === 1 ? '' : 's')
```

Comment:
I think this is simpler, but [ngettext](https://www.php.net/manual/en/function.ngettext.php) is a thing in PHP.  I like this better, but I guess ngettext is more idiomatic?

Reply:
The `$deleteCount` probably will be `0`